src/main/scala/Core/CSR.scala

```diff
@@ -119,22 +119,33 @@ class CSR(implicit val conf: FlexpretConfiguration) extends Module {
     CSR_W -> io.rw.data_in
   ))
 
+  // Check whether the write does not make any alterations; i.e. it is a fake write
```

Comment:
This is a good addition. Is there a simple way to add a Chisel test to verify that it works?